### PR TITLE
[T880] Travis CI- Configuration.

### PR DESCRIPTION
**Pull request by YogeshMahera-SerpentCS**
Task #[880](https://www.quartile.co/web#id=880&action=771&model=project.task&view_type=form&menu_id=505)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,33 +1,105 @@
...
+    # Other env possibilities:
+    # use this to install a standalone database to export .pot files
+    # - MAKEPOT="1"
+    # either use the two lines above or the two below. Don't change the default if
+    # it's not necessary (it is only necessary if modules in your repository can't
+    # be installed in the same database. And you get a huge speed penalty in your
+    # tests)
+    #  - TESTS="1.0" ODOO_REPO="odoo/odoo" UNIT_TEST="1"
+    #  - TESTS="1.0" ODOO_REPO="OCA/OCB" UNIT_TEST="1"
+    # QRTL: Add following tests the installation of all modules including those are excluded
```
Comment (TimLai125) [inline .travis.yml:72]: Comment out L72-L78

File: .travis.yml [hunk 2]
```diff
@@ -1,33 +1,105 @@
...
+    # by the `pov-wkhtmltopdf` repo.
+    - PHANTOMJS_VERSION="latest"
+    # The above line controls the PhantomJS version that is used for JS testing.
+    #   It is not necessary to include this value unless you are altering the default.
+    #   Use `OS` to skip the PhantomJS upgrade & use the system version instead.
+    - WEBSITE_REPO="1"
+    # Use the above line to install dependencies that are required for website repos:
+    # * SASS & Bootstrap-SASS
+    # * Compass
+    - EXCLUDE="partner_category_hierarchy,website_forum_visibility"
```
Comment (TimLai125) [inline .travis.yml]: We do not need exclude for nrq-custom

**Review by TimLai125 — CHANGES_REQUESTED** [review]

**Comment by YogeshMahera-SerpentCS** [comment]
Hi @TimLai125 Changes Amended

**Comment by YogeshMahera-SerpentCS** [comment]
Hi @TimLai125 It's green can you check this?

**Pull request merged**